app/logic/Passkey.scala

```diff
@@ -154,12 +185,104 @@ object Passkey {
       case exception =>
         Failure(
           JanusException(
-            userMessage = "Bad arguments for verification request",
+            userMessage = "Bad arguments for registration verification request",
             engineerMessage =
-              s"Bad arguments for verification request: ${exception.getMessage}",
+              s"Bad arguments for registration verification request: ${exception.getMessage}",
             httpCode = BAD_REQUEST,
             causedBy = Some(exception)
           )
         )
     }
+
+  /** Parses the authentication response from the browser. Call this when the
+    * user has authenticated to the browser using a passkey.
+    *
+    * See
+    * [[https://webauthn4j.github.io/webauthn4j/en/#webauthn-assertion-verification-and-post-processing]].
+    *
+    * @param jsonResponse
+    *   Json response from browser containing authentication data.
+    */
+  def parsedAuthentication(
+      jsonResponse: String
+  ): Try[AuthenticationData] =
+    Try {
+      webAuthnManager.parseAuthenticationResponseJSON(jsonResponse)
+    } recoverWith {
```

Comment:
This syntax isn't encouraged in modern Scala, although it does work. [The style guide](https://docs.scala-lang.org/style/method-invocation.html#higher-order-functions) recommends using `.` for method calls.

Reply:
Fixed in a26c9d6699bdad934a5460eacedb300ff2ab78be